marlowe-playground-server/app/PSGenerator.hs

```diff
@@ -152,12 +143,14 @@ myBridge :: BridgePart
 myBridge =
     -- See Note [GistID hotfix]
     gistIdBridge <|>
-    PSGenerator.Common.aesonBridge <|>
-    PSGenerator.Common.containersBridge <|>
-    PSGenerator.Common.languageBridge <|>
-    PSGenerator.Common.ledgerBridge <|>
-    PSGenerator.Common.servantBridge <|>
-    PSGenerator.Common.miscBridge <|>
+    -- FIXME: Just commenting these, not removing until I make SCP-4726 to see that it
+    --        does not affect the entire build
+    -- PSGenerator.Common.aesonBridge <|>
+    -- PSGenerator.Common.containersBridge <|>
+    -- PSGenerator.Common.languageBridge <|>
+    -- PSGenerator.Common.ledgerBridge <|>
+    -- PSGenerator.Common.servantBridge <|>
+    -- PSGenerator.Common.miscBridge <|>
```

Comment:
Why comment out instead of deleting? Is this to be replaced with something else down the road?